docs/setup.md

```diff
@@ -1,20 +1,65 @@
-| [Home](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/README.md) | 
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/release/1.0.1/README.md) | 
 |--------------------------------------------| 
  
 # Installation 
-1. To install a solution pack, click **Content Hub** > **Discover**.    
-2. From the list of solution pack that appears, search for and select **GDPR Framework**. 
-3. Click the **GDPR Framework** solution pack card.    
-4. Click the **Install** button on the bottom to begin installation. 
+1. To install a solution pack, click **Content Hub** > **Discover**.
+2. From the list of solution packs that appear, search for and select **GDPR Framework**.
+3. Click the **GDPR Framework** solution pack card.
+4. Click the **Install** button on the bottom to begin the installation.
 
 ## Prerequisites
 
 The **GDPR Framework** solution pack depends on the following solution packs.
 
-| Solution Pack Name | Purpose                                                     |
-|:-------------------|:------------------------------------------------------------|
-| SOAR Framework     | Required for Incident Response modules and Action playbooks |
+| Solution Pack Name    | Purpose                                                                                                                                                                                           |
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SOAR Framework v2.0.0 | <ol><li>Required for Incident Response modules and Action playbooks</li><li>Required for "Was personal data affected?" field which acts as a trigger point for GDPR Framework Playbooks</li></ol> |
```

Comment:
It is coming in single line instead of coming in next line
![image](https://user-images.githubusercontent.com/24773177/193516690-1bb8fe3a-e868-45ae-a7c6-902244501327.png)